server.js

```diff
@@ -1,35 +1,86 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import netflixTitles from "./data/netflix-titles.json";
+import listEndpoints from "express-list-endpoints"; 
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/Cluster0";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
+const port = process.env.PORT || 9000;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+const { Schema } = mongoose;
+
+const movieSchema = new mongoose.Schema({
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: [String],
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: [String],
+  description: String
+});
+
+const Movie = mongoose.model("Movie", movieSchema);
+
+
+
+if (process.env.RESET_DB) {
+  const resetDatabase = async () => {
+    await Movie.deleteMany();
+    netflixTitles.forEach((singleTitle) => {
+      const newMovie = new Movie(singleTitle);
+      newMovie.save()
+    })
+  }
+  resetDatabase();
+}
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+
+app.get("/", (req, res) => {  // this is the root route
+res.send(listEndpoints(app));
```

Comment:
It looks from your endpoints that you only created one endpoint, called ‘movies’, but that endpoint doesn’t show all the movies, without the use of filtering the data down using queries.